src/components/SpruceForm/FieldTemplates/index.tsx

```diff
@@ -50,4 +50,5 @@ const DefaultFieldContainer = styled.div<{ border?: "top" | "bottom" }>`
   ${({ border }) =>
     border &&
     `border-${border}: 1px solid ${gray.light1}; padding-${border}: ${size.s};`}
+  width: 100%;
```

Comment:
I noticed this might be causing a slight glitch with the variables input

<img width="772" alt="Screenshot 2023-09-07 at 10 08 05 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/c9d3f693-03cb-4e1e-b22f-3218767d7ccc">